codegen/src/main/java/io/improbable/keanu/codegen/python/ParamStringProcessor.java

```diff
@@ -0,0 +1,28 @@
+package io.improbable.keanu.codegen.python;
+
+import com.sun.javadoc.ConstructorDoc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.google.common.base.CaseFormat.LOWER_UNDERSCORE;
+import static com.google.common.base.CaseFormat.UPPER_CAMEL;
+
+class ParamStringProcessor {
+    static Map<String, String> getNameToCommentMapping(ConstructorDoc constructorDoc) {
+        String rawComment = constructorDoc.getRawCommentText();
+        String[] rawCommentLines = rawComment.split("\\r?\\n");
+        Map<String, String> nameToCommentMapping = new HashMap<>();
+        for (String commentLine : rawCommentLines) {
+            if (!commentLine.contains("@param")) {
```

Comment:
It looks like `constructorDoc.tags("@param")` gives you what you want in a cleaner way.